packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -233,6 +249,23 @@ rm -f %{buildroot}%{kernel_libdir}/build %{buildroot}%{kernel_libdir}/source
 ln -sf %{_usrsrc}/kernels/%{version} %{buildroot}%{kernel_libdir}/build
 ln -sf %{_usrsrc}/kernels/%{version} %{buildroot}%{kernel_libdir}/source
 
+# Ensure that each required FIPS module is loaded as a dependency of the
+# check-fips-module.service. The list of FIPS modules is different across
+# kernels but the check is consistent: it loads the "tcrypt" module after
+# the other modules are loaded.
+mkdir -p %{buildroot}%{_cross_unitdir}/check-fips-modules.service.d
+i=0
+for fipsmod in $(cat %{_sourcedir}/fipsmodules-%{_cross_arch}) ; do
+[ "${fipsmod}" == "tcrypt" ] && continue
+drop_in="$(printf "%03d\n" "${i}")-${fipsmod}.conf"
+cat <<EOF > %{buildroot}%{_cross_unitdir}/check-fips-modules.service.d/"${drop_in}"
+[Unit]
+Requires=fips-modprobe@${fipsmod}.service
+After=fips-modprobe@${fipsmod}.service
+EOF
```

Comment:
Why not a `drop-in.conf.in` and `sed` the replacement?

Reply:
> Why not a drop-in.conf.in and sed the replacement?

I can do that. It's more consistent with what we do in other specs, plus it avoids the awkward indentation problem of heredocs.